Added default ansible variables

lukehoersten opened the pull request:
Also made variables for user and other things.

I enabled pruning by default for people to get started faster.

lukehoersten committed:
Added default ansible variables

Also made variables for user and other things.

mdunnio commented on ansible/roles/raspbian/bitcoin/templates/bitcoin.conf.j2:
```diff
@@ -7,7 +7,8 @@
 daemon=1
-txindex=1
+#txindex=1
+prune=550
```
This should be left as is. Obviously, this is a highly opinionated configuration and users of this role will need to configure what they want.

lukehoersten committed:
added prune flag

Prune defaults to false.

lukehoersten committed:
Keep config with bitcoind data dir

The default place to look for the config is the data dir. It's good pratice to keep the config with the chainstate etc.
